sites/hashdev/src/_pages/blog/4_announcing-error-stack.mdx

```diff
@@ -0,0 +1,283 @@
+---
+authorPhoto: blog/4_announcing-error-stack/tim.jpg
+postPhoto: blog/4_announcing-error-stack/ErrorStack_Cover.jpg
+title: "Announcing error-stack"
+subtitle: "A context-aware error library with arbitrary attached user data"
+author: "Tim Diekmann"
+jobTitle: "Platform Engineer at HASH"
+date: "2022-06-10"
+---
+
+We are proud to announce a new error library for Rust: `error-stack` - a context-aware error library with arbitrary attached user data. **[View the crate on GitHub >](https://github.com/hashintel/hash/tree/main/packages/libs/error-stack)**
```

Comment:
I've just looked at the newest deployment and the opening paragraph looks a little strange. 

Perhaps we should put a line-break for "View the crate on Github".

Reply:
I agree. Opened as a suggestion: https://github.com/hashintel/hash/pull/628/files#r894458618